R/TealAppDriver.R

```diff
@@ -407,6 +407,14 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
     #' @return Nothing. Opens the underlying teal app in the browser.
     open_url = function() {
       browseURL(self$get_url())
+    },
+    #' @description
+    #' Wait until an output value is not one of `ignored` values `timeout` is reached.
+    #' This function will only access output value and its wrapper around `wait_for_value` method.
+    #' @param output_id (character) The shiny output id to get the value from.
+    wait_for_ouput_value = function(output_id, ...) {
+      checkmate::check_string(input_id)
+      self$wait_for_value(output = sprintf("%s-%s", self$active_module_ns(), output_id), ...)
```

Comment:
- What you do you think about renaming the method to explicitly say that this is for the active module output value?
- Should we check if `...` contains `export`, `output` or `input` parameter and throw an error if it does?

Reply:
OR lets create a wrapper for `wait_for_value` function that checks if the function contains export, output or input parameter and assigns it accordingly.

WDYT?

Reply:
So it would have `id` and then prepend to the correct? Sounds good.

`wait_for_active_module_value` as name?